### DISCUSSION TITLE
Licensing 2, 4, ... Servers

### DISCUSSION THREAD

**Discussion by taspeotis** [datalust/seq-tickets#1174] [Help]
Hi,

How do we license Seq installation numbers that aren't perfectly covered by Team (1 installation) or Enterprise (3 installations) without going up to Partner?

Presently we have two to license (prod/dev) which seems like the most straightforward way would be to buy two Team installations. But let's say we go four then do we have to mix-and-match one Enterprise license with one Team license? I note one Enterprise license and one Team license is marginally cheaper than four Team licenses individually so there is some incentive to avoid just buying one Team license per install.

Sorry if this is answered on the pricing page (https://datalust.co/pricing) but it did not seem clear to me.

Thanks in advance.

**Comment by nblumhardt** [comment]
Hi Todd, thanks for getting in touch! 1 &times; Enterprise + 3 &times; Team, or 4 &times; Team would both be fine. If you need more environments and it starts to get out of hand for you, we can help with a custom quote - drop us a line via `support@` any time, in that case :-)

All the best,
Nick

**Reply by taspeotis** [reply]
Yeah it dawned on me from reading the Q&A that I could ask for a quote. Thanks for the quick response.
